### PR TITLE
Update tsne widget endpoint with umap and tsne

### LINKED ISSUE #125: Update the tSNE widget with the default plot method and parameterisation

### PR DESCRIPTION
**Pull request by lingyun1010**
We need to add `plotTypesAndOptions` to the endpoint to make sure the widget component is compilable with the latest tsne plot component.

### CODE REVIEW

File: app/src/main/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentMetadataController.java [hunk 1]
```diff
@@ -38,7 +42,7 @@ public String getTSnePlotMetadata(@PathVariable String experimentAccession,
 
         return GSON.toJson(
                 ImmutableMap.of(
-                        "perplexities", experimentPageContentService.getPerplexities(experiment.getAccession()),
+                        "plotTypesAndOptions", GSON.toJsonTree(tsnePlotSettingsService.getAvailablePlotTypesAndPlotOptions(experimentAccession)),
                         "metadata", experimentPageContentService.getMetadata(experiment.getAccession()),
```
Comment (upendrakumbham) [inline app/src/main/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentMetadataController.java:47]: get `accession` from the experiment like the previous code base.

Comment (upendrakumbham) [reply app/src/main/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentMetadataController.java:47]: Did you try running the WIT test after this change?

Comment (lingyun1010) [reply app/src/main/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentMetadataController.java:47]: We don't have a WIT for this controller.

**Review by upendrakumbham — APPROVED** [review]
LGTM!

**Pull request merged**